nemo/export/tensorrt_llm.py

```diff
@@ -317,6 +325,115 @@ def export(
         if load_model:
             self._load()
 
+    def build(
```

Comment:
How is this different then the export? It looks similar to export. If yes, can we merge this function with the export function? We might add a couple of parameters to control anything specific with refit.

Reply:
For instance: https://github.com/NVIDIA/NeMo/blob/a973ef529f98c90be5825fac4bcaca869f25e294/nemo/export/tensorrt_llm.py#L243
In export a single rank creates the engines in a loop, but in build all gpus create engines simultaneously (because the distributed environment has already been created)